src/async_fuse/memfs/dist/etcd.rs

```diff
@@ -60,7 +61,7 @@ pub async fn get_node_ip_and_port(
             format!("Fail to get Node Ip address and port information, node_id:{node_id}")
         })?;
 
-    if let Some(ip_and_port) = ip_and_port {
+    if let Some((ip_and_port, _v)) = ip_and_port {
```

Comment:
If we usually drop the version, why don't we provide a function to drop it and return the value alone?

Reply:
Then we need a fn get_at_most_one_value and fn get_at_most_one_value_without_version. I think the difference is very small, and the copy of version cost very little.

Reply:
Changed to related functions to fn xxxx and fn xxxx_with_version, and the default xxxx returns withouts version.